intro-to-tests/tdd-exercise.instructor.md

```diff
@@ -1,15 +1,17 @@
 # Instructor: TDD Exercise
 
-
+## We are no longer using replit for livecodes and activities
 Replit Activity: [https://replit.com/@adadev/tdd-exercise#README.md](https://replit.com/@adadev/tdd-exercise#README.md)
 
 Replit Solution: [https://replit.com/@adadev/tdd-exercise-solution](https://replit.com/@adadev/tdd-exercise-solution)
 
 Old Repo: [https://github.com/AdaGold/tdd-exercise](https://github.com/AdaGold/tdd-exercise), not used because students haven't learned how to collaborate in Git yet.
 
+Current Repo (introduced in C22): [https://github.com/Ada-Activities/tdd-exercise](https://github.com/Ada-Activities/tdd-exercise)
+
 Solution Branch in Old Repo: [https://github.com/AdaGold/tdd-exercise/blob/solution-python/tests/test_blackjack_score.py](https://github.com/AdaGold/tdd-exercise/blob/solution-python/tests/test_blackjack_score.py)
```

Comment:
How do you feel about moving the current link to the top of the page so it's the first thing folks see and removing the AdaGold repo links?

```suggestion
Current Repo (introduced in C22): [https://github.com/Ada-Activities/tdd-exercise](https://github.com/Ada-Activities/tdd-exercise)
- Solution is on the `solution` branch in this repo

## We are no longer using replit for livecodes and activities
Replit Activity: [https://replit.com/@adadev/tdd-exercise#README.md](https://replit.com/@adadev/tdd-exercise#README.md)

Replit Solution: [https://replit.com/@adadev/tdd-exercise-solution](https://replit.com/@adadev/tdd-exercise-solution)
```